examples/vgrade.py

```diff
@@ -47,7 +47,7 @@ def stopwatch(message=None):
         return
     now = pg.time.get_ticks()
     runtime = (now - timer) / 1000.0 + 0.001
-    print("%s %s %s" % (message, runtime, ("seconds\t(%.2ffps)" % (1.0 / runtime))))
+    print("{} {} {}".format(message, runtime, ("seconds\t(%.2ffps)" % (1.0 / runtime))))
```

Comment:
I think this could be further improved with some manual formatting

Reply:
Pyupgrade is [intentionally timid](https://github.com/asottile/pyupgrade#f-strings) in converting to f-strings if it would harm readability.
[Flynt](https://pypi.org/project/flynt/) is more aggressive if we want to try that on a _separate_ follow-up PR.

Reply:
I personally prefer linters to not be too aggressive, and I believe such changes are better made manually.
But I'm in for testing flynt after this PR is merged, to minimise merge conflicts too

Reply:
```suggestion
    print("{message} {runtime} {'seconds\t(%.2ffps)' % (1.0 / runtime)}")
```

Reply:
I think that can further be simplified to
`print(f"{message} {runtime} seconds\t{(1.0 / runtime):.2f}fps")`

Reply:
I think this has not been resolved yet... although it was marked as so

Reply:
It is now, thanks!